microshift_configuring/microshift-using-config-tools.adoc

```diff
@@ -5,10 +5,8 @@ include::_attributes/attributes-microshift.adoc[]
 :context: microshift-configuring
 toc::[]
 
-{product-title} uses a YAML configuration file to execute commands.
-
-//include::modules/microshift-config-cli-manifests.adoc[leveloffset=+1]
+{product-title} uses a YAML configuration file.
```

Comment:
Hhmm, is is more correct and descriptive to say something like, "A YAML file customizes {product-title} instances with your preferences, settings, and parameters."

Reply:
It's certainly more detailed. :-)

Done.